### DISCUSSION TITLE
Is it possible to have more records per page?

### DISCUSSION THREAD

**Discussion by 2otherjobs** [poweradmin/poweradmin#485] [Ideas]
I have been a user of poweradmin for about 6 years.  Thank you very much for keeping it alive!

In one domain, I have 120 records.  With older versions (say, 2.2.2), the entire list spanned three pages and was easy to scan/review/edit.  The newest version 3.4.0 looks great, and I appreciate the improvements 'under the hood'.  However, 120 records takes 13 pages - The most my web browser showed was 10 records per page.  Could you add an option to enable selection for the number of records per page?  I really appreciated the record density of the older versions.

**Comment by edmondas** [comment]
Glad to hear you've been using this software for so long.

It is possible to control the item per page using the option `$iface_rowamount`,  but this is global setting. Future plans include the ability for users to select items on each page themselves.

**Comment by 2otherjobs** [comment]
I changed that setting, which worked as you said.  Thanks for your reply.

Another question: For each DNS record,  the edit and delete boxes are stacked over/under rather than side by side.  I am working with a high-resolution laptop screen.  This forces each record to use two lines instead of one (the second line only contains the ‘Delete’ box, and nothing else).
If the boxes were side by side, the records could fit onto one line and more would be visible at one time - more ‘record density’ per page.  Is there another setting I could adjust to put each record on one line instead of consuming two lines?

Off-topic Comment:  I am not a Linux user.  I use poweradmin in FreeBSD jails, and it works smoothly.  Very easy to install and maintain.

**Comment by edmondas** [comment]
Would it be possible to provide a screenshot with anonymized data for better understanding? In the last version, I tried to address this, but there might have been places where I overlooked it.

By the way, FreeBSD is a really nice operating system, I used it a lot in the past because of its simplicity and power.

**Comment by 2otherjobs** [comment]
As requested.  Hope this help!  I have tried adjusting screen resolutions, etc.   The Edit/Delete boxes remain ’stacked’ rather than 'side-by-side'.






By the way, the list of domains does NOT have this problem.  The domain name list has the Edit and Delete. boxes side by side.  Good work!

Tom

**Reply by edmondas** [reply]
I'm not sure why the screenshot isn't available, so maybe you can specify which php script is responsible for this incorrect behavior (edit, edit_record or other)?

**Comment by 2otherjobs** [comment]
Here’s another try as sending you the screenshot:

￼

As for which php script causes this behavior, I am afraid I do not know.  This screenshot comes from the list of records for a particular domain.  Other domains look the same.  

Interestingly, the page showing the domain list does NOT put the Edit and Delete boxes above and below on two separate lines.  Instead, the Edit and Delete boxes are side-by-side.  This makes the Zone List page more efficient - one line per zone (domain) rather than two.

I can try to transmit the image another way if this attempt fails.

Tom

**Reply by edmondas** [reply]
The same... The screenshot is not visible. Please send it by email. It can be found at the bottom of the project's page (footer).

**Comment by edmondas** [comment]
The edit.html file has been modified (check line 107) -https://github.com/poweradmin/poweradmin/commit/2c42e7ad0c3bc9aa6ed0bf9e2aa1da8d04ccec50

**Reply by mxdabc** [reply]
> The edit.html file has been modified (check line 107) -[2c42e7a](https://github.com/poweradmin/poweradmin/commit/2c42e7ad0c3bc9aa6ed0bf9e2aa1da8d04ccec50)

Did the latest Release cancel it again?

**Reply by edmondas** [reply]
@mxdabc The pagination and layout features are not cancelled - current release has basic pagination support and the upcoming release includes full user-selectable pagination (10, 20, 50 or 100 records per page) with preferences remembered between sessions, expanding beyond the original request.

What specific issue are you experiencing with the current version? This would help me understand what might be causing confusion about the feature status.

**Reply by mxdabc** [reply]
> What specific issue are you experiencing with the current version? This would help me understand what might be causing confusion about the feature status.

![image](https://github.com/user-attachments/assets/83452057-d378-4bf4-ac0f-bed336ffa1b0)

I downloaded the poweradmin program from the Release page on Github.
If there are more than 10, you need to click on the next page, and there is no drop-down menu to choose how many pages can be displayed.

**Reply by edmondas** [reply]
> I downloaded the poweradmin program from the Release page on Github. If there are more than 10, you need to click on the next page, and there is no drop-down menu to choose how many pages can be displayed.

This feature is currently included in the unreleased 4.0.0 version, which is still under testing. However, I can add this pagination to the 3.9.x branch and make a release. I'll likely do this after conducting some checks.

**Reply by mxdabc** [reply]
> This feature is currently included in the unreleased 4.0.0 version, which is still under testing. However, I can add this pagination to the 3.9.x branch and make a release. I'll likely do this after conducting some checks.

Thanks for the answer, I decided that I would download the 4.x version.

**Comment by 2otherjobs** [comment]
Thank you👍Sent from my iPhoneOn Jan 8, 2023, at 1:00 PM, Edmondas Girkantas ***@***.***> wrote:﻿
The edit.html file has been modified (check line 107) -2c42e7a

**Comment by edmondas** [comment]
@2otherjobs I've recently implemented a feature allowing users to select how many items display per page. This enhancement will address the problem you've reported and will be included in our upcoming 4.0.0 release.

<img width="1319" alt="image" src="https://github.com/user-attachments/assets/bfc1dfe1-6078-4066-864b-29b1cbd2a163" />

**Comment by 2otherjobs** [comment]
Thank you for making this possible.  On Apr 14, 2025, at 4:15 AM, Edmondas Girkantas ***@***.***> wrote:﻿
@2otherjobs I've recently implemented a feature allowing users to select how many items display per page. This enhancement will address the problem you've reported and will be included in our upcoming 4.0.0 release.
image.png (view on web)
